test/L1Escrow.t.sol

```diff
@@ -154,7 +154,92 @@ contract EscrowTest is Setup {
         assertEq(vault.balanceOf(address(mockEscrow)), 0);
     }
 
-    function test_bridgeAsset_maxDepositLimit(uint256 _amount) public {
+    function test_bridgeAsset_escrowDepositLimit(uint256 _amount) public {
+        _amount = bound(_amount, minFuzzAmount, maxFuzzAmount);
```

Comment:
can _amount be 0?

Reply:
yes, will add a section to show that

Reply:
Test using a 0 deposit limit is here https://github.com/yearn/yearn-stb/pull/17/files#diff-52b3b649a24d8bf5f33c53bf9546c7aee0a6133a5503d8525a1a1e5dab36f656R170